src/Datasource/Connection.php

```diff
@@ -201,6 +201,6 @@ protected function _log($context)
         $data = json_encode($logData, JSON_PRETTY_PRINT);
         $loggedQuery = new LoggedQuery();
         $loggedQuery->query = $data;
-        $this->_logger->log('debug', $loggedQuery);
```

Comment:
@markstory I always get:

2016-06-08 23:58:33 Error: [TypeError] Argument 1 passed to DebugKit\Database\Log\DebugLog::log() must be an instance of Cake\Database\Log\LoggedQuery, string given, called in /home/damiano/www/test.local/vendor/cakephp/elastic-search/src/Datasource/Connection.php on line 2

after **$query->all()**